crates/refunder/src/lib.rs

```diff
@@ -6,14 +6,25 @@ pub mod submitter;
 use contracts::CoWSwapEthFlow;
 use ethcontract::{Account, PrivateKey};
 use refund_service::RefundService;
-use shared::http_client::HttpClientFactory;
+use shared::{http_client::HttpClientFactory, metrics::LivenessChecking};
 use sqlx::PgPool;
-use std::time::Duration;
+use std::{
+    sync::{Arc, RwLock},
+    time::{Duration, Instant},
+};
 
-const SLEEP_TIME_BETWEEN_LOOPS: u64 = 30;
+const SLEEP_TIME_BETWEEN_LOOPS: u64 = 1;
+const SECONDS_FROM_LAST_LOOP_BEFORE_UNHEALTHY: u64 = 4 * SLEEP_TIME_BETWEEN_LOOPS;
```

Comment:
Ubernit: I would make these `const`s into `Duration`s:

```suggestion
const LOOP_INTERVAL: Duration = Duration::from_secs(1);
const DELAY_FROM_LAST_LOOP_BEFORE_UNHEALTHY: Duration = LOOP_INTERVAL.saturating_mul(4);
```

Reply:
Thanks for pointing out `saturating_mul`. :slightly_smiling_face: